utils/server_config.json

```diff
@@ -20,7 +20,7 @@
       "host": "",
       "port": "",
       "model_endpoint": "/ecommerce_bot",
-      "model_args_names": ["query", "state"]
+      "model_args_names": ["query", "state", "history"]
```

Comment:
Agent should accept `query` and `session_id`

Reply:
fixed

Reply:
Why? This entry for `skill`, not `agent`.

Reply:
Then arguments order was wrong